theme/static/js/hs_resource_table.js

```diff
@@ -24,9 +24,10 @@ var filtersLoaded = []
 var spinner = $('<i class="fa fa-spinner fa-pulse fa-lg my-res-filter-spinner" style="z-index: 1;"></i>')
 
 $(document).ready(function () {
-    setInitialFilters()
+    const isMyResPage = ROUTE_NAME.indexOf("my_resources") > -1;
```

Comment:
Since we are now using the route name we can do a simple string equality check instead of `indexOf`

Reply:
thanks, updated that.